src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProviderContentTypes.java

```diff
@@ -39,15 +39,47 @@
 import com.synopsys.integration.blackduck.notification.content.VulnerabilitySourceQualifiedId;
 
 public class BlackDuckProviderContentTypes {
+    //common fields
+    public static final String JSON_FIELD_CONTENT = "content";
+    public static final String JSON_FIELD_PROJECT_NAME = "projectName";
+    public static final String JSON_FIELD_PROJECT_VERSION_NAME = "projectVersionName";
+    public static final String JSON_FIELD_PROJECT_VERSION = "projectVersion";
+    public static final String JSON_FIELD_COMPONENT_NAME = "componentName";
+    public static final String JSON_FIELD_COMPONENT = "component";
+    public static final String JSON_FIELD_COMPONENT_VERSION_NAME = "componentVersionName";
+    public static final String JSON_FIELD_COMPONENT_VERSION = "componentVersion";
+
+    public static final String CONFIG_MAPPING_CONFIGURED_PROJECTS = "configuredProjects";
+
+    // policy fields
+    public static final String JSON_FIELD_COMPONENT_VERSION_STATUSES = "componentVersionStatuses";
+    public static final String JSON_FIELD_POLICY_INFOS = "policyInfos";
+    public static final String JSON_FIELD_POLICY_NAME = "policyName";
+    public static final String JSON_FIELD_POLICY = "policy";
+    public static final String JSON_FIELD_FIRST_NAME = "firstName";
+    public static final String JSON_FIELD_LAST_NAME = "lastName";
+
+    // vulnerability fields
+    public static final String JSON_FIELD_AFFECTED_PROJECT_VERSIONS = "affectedProjectVersions";
+    public static final String JSON_FIELD_VERSION_NAME = "versionName";
+    public static final String JSON_FIELD_NEW_VULNERABILITY_IDS = "newVulnerabilityIds";
+    public static final String JSON_FIELD_UPDATED_VULNERABILITY_IDS = "updatedVulnerabilityIds";
+    public static final String JSON_FIELD_DELETED_VULNERABILITY_IDS = "deletedVulnerabilityIds";
+
+    // labels
     public static final String LABEL_COMPONENT_NAME = "Component";
     public static final String LABEL_COMPONENT_VERSION_NAME = "Component Version";
     public static final String LABEL_POLICY_NAME = "Policy";
     public static final String LABEL_PROJECT_NAME = "Project";
     public static final String LABEL_PROJECT_VERSION_NAME = "Project Version";
+    public static final String LABEL_POLICY_OVERRIDE_FIRST_NAME = JSON_FIELD_FIRST_NAME;
```

Comment:
Keep in mind, the label is the thing that shows up in the channel message, so case might matter for consistency:
e.g.
firstName: Bob
vs
Policy Overrider: Bob

Reply:
This concern is currently addressed in the policy collector object